eng/Version.Details.xml

```diff
@@ -30,6 +30,10 @@
       <Uri>https://github.com/aspnet/xdt</Uri>
       <Sha>d4d088b6a9c793525b1a27a119cb66ba4587bb39</Sha>
     </Dependency>
+    <Dependency Name="System.CommandLine" Version="2.0.0-beta4.23307.1">
```

Comment:
Additionally, wouldn't it be best to actually upgrade NuGet to be on the same version as .NET?

Reply:
Probably, yes, However, there isn't a single version used by other repos. There are at least 3 different versions used today across 11 repos that consume this package. `arcade` uses the same version as `nuget.client`. The version is 6 months old.

`installer` repo uses the most recent version, from couple weeks ago. I'll update `nuget.client` to use the same.

While other repos can use the dependency flow to get on latest, `nuget.client` repo does not have that support.

Reply:
Fixed with https://github.com/NuGet/NuGet.Client/pull/5625/commits/a06168918fc250a2088a0ce4f18808bd98f6d1d3